tests/Doctrine/Migrations/Tests/DependencyFactoryTest.php

```diff
@@ -91,6 +91,7 @@ public function testConnection() : void
 
         self::assertSame($this->connection, $di->getConnection());
         self::assertFalse($di->hasEntityManager());
+        self::assertTrue($di->isFrozen());
```

Comment:
Testing explicitly if the `DependencyFactory`. Does this mean that in the other tests it should be checked if it isn't frozen in case this is relevant for regressions?